client/state/themes/selectors/get-theme-demo-url.js

```diff
@@ -15,5 +15,12 @@ import 'calypso/state/themes/init';
  */
 export function getThemeDemoUrl( state, themeId, siteId ) {
 	const theme = getCanonicalTheme( state, siteId, themeId );
+	//Temp. workaround to show an accurate theme demo site for Storefront
```

Comment:
It might help to link to the [github comment](https://github.com/Automattic/wp-calypso/issues/37658#issuecomment-843426848) which contains the reason why we did this, so the next person knows what to look out for when they undo this.